package.json

```diff
@@ -7,7 +7,7 @@
     "@mantine/core": "^7.8.0",
     "@mantine/form": "^7.8.0",
     "@mantine/hooks": "^7.8.0",
-    "@metabase/embedding-sdk-react": "^0.53.4",
+    "@metabase/embedding-sdk-react": "^0.53.6",
```

Comment:
Reminder that we'll have to use 53.7 with the fix from Tim, as 53.6 does not support click actions in dashboards.